src/css/components/menu/_menu-inline.scss

```diff
@@ -13,6 +13,7 @@
 // Hover state
 .video-js .vjs-menu-button-inline:hover,
 .video-js .vjs-menu-button-inline:focus,
+.video-js .vjs-menu-button-inline[aria-expanded="true"],
```

Comment:
My understanding is that attribute queries are still slow in most browsers. Is there a class we could hang this off of?